<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Geo</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonCube">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cube</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSphere">
      <property name="text">
       <string>Sphere</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonTorus">
      <property name="text">
       <string>Torus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
